docs/guides/websites/hosting/secure-website-lets-encrypt-acme-sh/index.md

```diff
@@ -22,17 +24,19 @@ external_resources:
 
 ## Introduction
 
-[acme.sh](https://acme.sh/) is a client application for ACME-compatible services, such as those used by [Let's Encrypt](https://letsencrypt.org/).
+[acme.sh](https://acme.sh/) is a client application for ACME-compatible services, such as those used by [Let's Encrypt](https://letsencrypt.org/). It is an alternative to the popular [Certbot](/docs/guides/quick-answers/websites/) application with two big benefits:
 
-It is an alternative to the popular Certbot application with two big benefits: it is written as pure shell scripting so it has no dependencies, and supports **a lot** more DNS providers.
+- It is written as pure shell scripting so it has no dependencies
+
+- It supports [**a lot** more DNS providers](https://github.com/acmesh-official/acme.sh/wiki/dnsapi).
 
 If you use Linode for your website's DNS, you can use acme.sh to obtain both single and wildcard SSL certificates, using Linode DNS as the domain ownership verification.
 
 ## Before You Begin
 
 1. Ensure that you have followed the [Getting Started](/docs/getting-started/) and the [Securing Your Server](/docs/security/securing-your-server/) guides.
 
-2. Decide which user account will issue and renew the certificates and switch to this user. Bear in mind if you are wanting to automatically restart a web server, or write certificates to a restricted folder, you will likely want to install acme.sh under root.
+2. Decide which user account will issue and renew the certificates and switch to this user. If you want to automatically restart a web server, or write certificates to a restricted folder, you will likely want to install acme.sh under root.
```

Comment:
Just simplifying the sentence structure a bit, which might make it easier to translate. We're making efforts to translate all guides in our library in the future, so hopefully a change like this helps with that